src/main/kotlin/com/expediagroup/openworld/sdk/core/configuration/ClientConfiguration.kt

```diff
@@ -21,11 +21,13 @@ interface ClientConfiguration {
     val key: String?
     val secret: String?
     val endpoint: String?
+    val requestTimeoutMillis: Long?
 
     /** Build a [RuntimeConfigurationProvider] from a [ClientConfiguration]. */
     fun toProvider(): RuntimeConfigurationProvider = RuntimeConfigurationProvider(
         key = key,
         secret = secret,
-        endpoint = endpoint
+        endpoint = endpoint,
+        requestTimeoutMillis = requestTimeoutMillis
```

Comment:
Let's rename it to `requestTimeout` over all the code, and name the parameter as `milliseconds` in the builders.

Reply:
Is not `requestTimeoutMillis` more intuitive for any developer to figure out that the value is measured with milliseconds when used all over the code?

Reply:
Yes, but the only way to provide this by the partner will be via our builder, which has the parameter name as `milliseconds` as I added in another comment. Both of them will tell the same info in a more elegant way (in addition, it's the common way to provide this value in milliseconds).

_Anyway, if you feel the other way around, go for it, it's not a big deal._